src/kilo.c

```diff
@@ -318,6 +318,40 @@ void editorDelRow(int at) {
     E.dirty++;
 }
 
+void editorDeleteLines(int brow, int erow) {
+  if (brow > erow)
+    SWAP(brow, erow);
+
+  int iter = erow - brow;
+  int uppos = E.cy + E.rowoff - iter;
```

Comment:
why the heck don't you call stuff `beginRow` and such. `uppos` means NOTHING to me. could just as well be `x`. Would you give it a longer name that describes wtf it is.